cura/Settings/ContainerManager.py

```diff
@@ -394,7 +583,34 @@ def _updateContainerNameFilters(self):
             name_filter = "{0} ({1})".format(mime_type.comment, suffix_list)
             self._container_name_filters[name_filter] = entry
 
-    # Factory function, used by QML
-    @staticmethod
-    def createContainerManager(engine, js_engine):
-        return ContainerManager()
+    def _getQualityContainers(self, quality_name, container_type = "quality_changes"):
```

Comment:
It seems somewhat illogical to have a function that says it gets quality containers, but secretly may return quality-changes containers. Why did you collate these two types of containers?